gui/settings/settings.go

```diff
@@ -166,6 +166,16 @@ func (s *Settings) SetShowEmptyGroups(val bool) {
 	s.setBooleanSetting("show-empty-groups", val)
 }
 
+// GetIgnoreUnknown returns the ignore-non-roster setting
```

Comment:
I think it's a bit weird to have part of this configuration in the configuration file, and another part of it in the gsettings - it makes more sense to have it in one single place.